js/app/interfaces/card.js

```diff
@@ -189,6 +189,15 @@ const Card = new Lang.Interface({
         'sequence': GObject.ParamSpec.uint('sequence', 'Sequence', 'Sequence',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             0, Object.keys(Sequence).length, Sequence.NONE),
+        /**
+         * Property: can-mark-read
+         * Whether to mark cards as having been read
+         *
+         */
+        'can-mark-read': GObject.ParamSpec.boolean('can-mark-read', 'Can mark read',
```

Comment:
No, because, for example, we don't want the checkmark to show up on list cards in travel app or prensa libre app